sdk/spring/azure-spring-boot-test-aad-b2c/src/test/java/com/azure/test/aad/b2c/selenium/AADB2CIT.java

```diff
@@ -24,48 +27,53 @@
+    }
 
     @Test
-    public void testSignIn() throws InterruptedException {
-        aadB2CSeleniumITHelper = new AADB2CSeleniumITHelper(DumbApp.class, Collections.emptyMap());
-        aadB2CSeleniumITHelper.signIn(AADB2CTestUtils.AAD_B2C_SIGN_UP_OR_SIGN_IN);
-        String name = aadB2CSeleniumITHelper.getName();
-        String userFlowName = aadB2CSeleniumITHelper.getUserFlowName();
-
+    public void testSignIn() {
+        String name = aadb2cSeleniumITHelper.getName();
+        String userFlowName = aadb2cSeleniumITHelper.getUserFlowName();
         Assert.assertNotNull(name);
         Assert.assertNotNull(userFlowName);
-        Assert.assertEquals(AADB2CTestUtils.AAD_B2C_SIGN_UP_OR_SIGN_IN, userFlowName);
+        Assert.assertEquals(AAD_B2C_SIGN_UP_OR_SIGN_IN, userFlowName);
     }
 
     @Test
-    public void testProfileEdit() throws InterruptedException {
-        aadB2CSeleniumITHelper = new AADB2CSeleniumITHelper(DumbApp.class, Collections.emptyMap());
-        aadB2CSeleniumITHelper.signIn(AADB2CTestUtils.AAD_B2C_SIGN_UP_OR_SIGN_IN);
-        String currentJobTitle = aadB2CSeleniumITHelper.getJobTitle();
+    public void testProfileEdit() {
+        aadb2cSeleniumITHelper.profileEditJobTitle(JOB_TITLE_A_WORKER);
+        String currentJobTitle = aadb2cSeleniumITHelper.getJobTitle();
         String newJobTitle = JOB_TITLE_A_WORKER.equals(currentJobTitle) ? JOB_TITLE_WORKER : JOB_TITLE_A_WORKER;
-        aadB2CSeleniumITHelper.profileEditJobTitle(newJobTitle);
-        String name = aadB2CSeleniumITHelper.getName();
-        String jobTitle = aadB2CSeleniumITHelper.getJobTitle();
-        String userFlowName = aadB2CSeleniumITHelper.getUserFlowName();
-
+        aadb2cSeleniumITHelper.profileEditJobTitle(newJobTitle);
+        String name = aadb2cSeleniumITHelper.getName();
+        String jobTitle = aadb2cSeleniumITHelper.getJobTitle();
+        String userFlowName = aadb2cSeleniumITHelper.getUserFlowName();
         Assert.assertNotNull(name);
         Assert.assertNotNull(jobTitle);
         Assert.assertEquals(newJobTitle, jobTitle);
         Assert.assertEquals(AADB2CTestUtils.AAD_B2C_PROFILE_EDIT, userFlowName);
     }
 
     @Test
-    public void testLogOut() throws InterruptedException {
-        aadB2CSeleniumITHelper = new AADB2CSeleniumITHelper(DumbApp.class, Collections.emptyMap());
-        aadB2CSeleniumITHelper.signIn(AADB2CTestUtils.AAD_B2C_SIGN_UP_OR_SIGN_IN);
-        String signInButtonText = aadB2CSeleniumITHelper.logoutAndGetSignInButtonText();
+    public void testLogOut() {
+        aadb2cSeleniumITHelper.logout();
+        String signInButtonText = aadb2cSeleniumITHelper.getSignInButtonText();
         Assert.assertEquals("Sign in", signInButtonText);
+        aadb2cSeleniumITHelper.destroy();
     }
 
     @After
-    public void quitDriver() {
-        aadB2CSeleniumITHelper.quitDriver();
+    public void aadb2cSeleniumITHelperDestroy() {
+        if (aadb2cSeleniumITHelper != null) {
```

Comment:
According to your code, the aadb2cSeleniumITHelper would never be null, so I guess we can skip this null check.